lib/src/list.dart

```diff
@@ -66,64 +73,73 @@ class RealmList<T extends Object> extends collection.ListBase<T> {
     }
   }
 
-  /// Sets the element at the specified index in the list.
   @override
   void operator []=(int index, T value) {
     RealmListInternal.setValue(handle, realm, index, value);
   }
 
-  /// Clears the collection in memory and the references
-  /// to the objects in this collection in Realm.
+  @override
```

Comment:
something is off with the doc comments. The doc comment on `clear` is deleted.

Reply:
oh ok, I remembered we talked about this.
In this case I believe we have to include the doc comment here since it has Realm specifics

Reply:
Does it have any realm specific behaviour? The fact that the items are not deleted just because they are removed from the list, is no different than a regular list.. Or am I missing something?